src/gui/gui.cc

```diff
@@ -1268,38 +1263,38 @@ in Configuration->Emulation, restart PCSX-Redux, then try again.)"));
     {
         unsigned counter = 0;
         for (auto& editor : m_mainMemEditors) {
-            if (editor.show) {
+            if (editor.m_show) {
                 ImGui::SetNextWindowPos(ImVec2(520, 30 + 10 * counter), ImGuiCond_FirstUseEver);
                 ImGui::SetNextWindowSize(ImVec2(484, 480), ImGuiCond_FirstUseEver);
-                editor.draw(g_emulator->m_mem->m_wram, 8 * 1024 * 1024, 0x80000000);
+                editor.draw(g_emulator->m_mem->m_wram, 8 * 1024 * 1024);
             }
             counter++;
         }
-        if (m_parallelPortEditor.show) {
+        if (m_parallelPortEditor.m_show) {
             ImGui::SetNextWindowPos(ImVec2(520, 30 + 10 * counter), ImGuiCond_FirstUseEver);
             ImGui::SetNextWindowSize(ImVec2(484, 480), ImGuiCond_FirstUseEver);
-            m_parallelPortEditor.draw(g_emulator->m_mem->m_exp1, 512 * 1024, 0x1f000000);
+            m_parallelPortEditor.draw(g_emulator->m_mem->m_exp1, 64 * 1024);
```

Comment:
Probably don't revert this size.

Reply:
Oops. This was my mistake during a merge conflict.

Reply:
You still need to fix it tho :P